okhttp/src/main/kotlin/okhttp3/internal/concurrent/TaskQueue.kt

```diff
@@ -76,22 +76,24 @@ class TaskQueue internal constructor(
   }
 
   /** Overload of [schedule] that uses a lambda for a repeating task. */
-  inline fun schedule(
```

Comment:
We may not want to land this, but are effectively blocked.  We want Kotlin 1.5.  We want graal support.  We probably want this inlined.  But we can't have all 3 unless we get a fix for https://github.com/oracle/graal/issues/3466.

Reply:
Oh that’s too bad.
https://github.com/square/okhttp/pull/6917